src/main/io/serial_cli.c

```diff
@@ -1007,21 +1007,27 @@ static void cliSerial(char *cmdline)
 static void cliAdjustmentRange(char *cmdline)
 {
     int i, val = 0;
+    float valf = 0.0f;
     char *ptr;
+    char buf[8];
 
     if (isEmpty(cmdline)) {
         // print out adjustment ranges channel settings
         for (i = 0; i < MAX_ADJUSTMENT_RANGE_COUNT; i++) {
             adjustmentRange_t *ar = &currentProfile->adjustmentRanges[i];
-            cliPrintf("adjrange %u %u %u %u %u %u %u\r\n",
+            cliPrintf("adjrange %u %u %u %u %u %u %u %u",
                 i,
                 ar->adjustmentIndex,
                 ar->auxChannelIndex,
                 MODE_STEP_TO_CHANNEL_VALUE(ar->range.startStep),
                 MODE_STEP_TO_CHANNEL_VALUE(ar->range.endStep),
                 ar->adjustmentFunction,
-                ar->auxSwitchChannelIndex
+                ar->auxSwitchChannelIndex,
+                ar->enableDirectMode
             );
+            cliPrintf("%s", ftoa(ar->valueRange.min, buf));
```

Comment:
Since there is only a single char buffer, it couldn't pass both min and max values simultaneously to the `cliPrintf()` call above.
That said, I suppose adding a second local char buffer to `cliAdjustmentRange` and just having the single call wouldn't hurt.